### DISCUSSION TITLE
'from pymilvus import connections, db' from ubuntu can't import db

### DISCUSSION THREAD

**Discussion by MoonEsther** [milvus-io/milvus#26660] [Q&A and General discussion]
from pymilvus import connections, db

pip3 install pymilvus==2.2.7
the same command in windows can import db
but can't import from ubuntu

**Comment by XuanYang-cn** [comment]
We don't support db in 2.2.7,  It's a new feature in pymilvus==2.2.9 6bb23e37715392f9398e5b289cc0a4f2667fc563

Perhaps you're running a newer pymilvus version in windows? try use the following to get the version of pymilvus
```python
>>> import pymilvus
>>> pymilvus.__version__
2.2.9
```
